Adamant/Services/HealthCheckWrapper.swift

```diff
@@ -94,16 +94,45 @@ class HealthCheckWrapper<Service, Error: HealthCheckableError> {
             .store(in: &subscriptions)
     }
     
+    func waitingRequest<Output>(
+        _ requestAction: @Sendable @escaping (Service, Node) async -> Result<Output, Error>
+    ) async -> Result<Output, Error> {
+        guard allowedNodes.isEmpty else {
+            return await request(requestAction)
```

Comment:
What if due to multithreading `allowedNodes` became empty right after checking?

It's better to remove the whole guard